src/test/java/de/undercouch/gradle/tasks/download/DownloadTaskPluginTest.java

```diff
@@ -214,6 +214,24 @@ public void downloadSingleURL() throws Exception {
         byte[] dstContents = FileUtils.readFileToByteArray(dst);
         assertArrayEquals(contents, dstContents);
     }
+
+    /**
+     * Tests if a single file can be downloaded from a URL
+     * only if the source file is newer
+     * @throws Exception if anything goes wrong
+     */
+    @Test
+    public void downloadSingleURLOnlyIfNewer() throws Exception {
```

Comment:
Very much appreciated! Take your time! It's an open-source project and many people depend on it so we really need to be careful here. Thank you so much!